data/saas/config/hubspot_config.yml

```diff
@@ -121,3 +118,37 @@ saas_config:
               configuration:
                 field: status
                 value: SUBSCRIBED
+    - name: users
+      requests:
+        read:
+          path: /settings/v3/users/
+          method: GET
+          query_params:
+            - name: limit
+              value: 100
+          param_values:
+            - name: placeholder
+              identity: email
+          postprocessors:
+            - strategy: unwrap
```

Comment:
We can change this `unwrap` postprocessor to the simpler `data_path: results` notation.

Reply:
thanks for catching this, changed it